Dockerfile.upgrade_analyzer

```diff
@@ -0,0 +1,30 @@
+FROM alpine:3.4
+
+RUN apk add --no-cache \
+    ruby=2.3.1-r0 \
+    ruby-dev=2.3.1-r0 \
+    build-base=0.4-r1 \
+    libffi-dev=3.2.1-r2 \
+    ruby-irb=2.3.1-r0 \
+    ca-certificates=20160104-r4 \
+    libxml2-dev=2.9.4-r0 \
+    libxslt-dev=1.1.29-r0
+
+RUN update-ca-certificates
+
+RUN echo 'gem: --no-rdoc --no-ri' >> /etc/gemrc && \
+    gem install \
+    bundler:1.13.1 \
+    rdoc:4.2.2 \
+    io-console:0.4.6
+
+COPY Gemfile Gemfile
+COPY Gemfile.lock Gemfile.lock
+RUN bundle install
+RUN mkdir -p /opt/ci
+
+COPY . /opt/ci
```

Comment:
How does this work for development within the container?  Shouldn't we mount the directory instead?